### DISCUSSION TITLE
Enum type in Postgresql

### DISCUSSION THREAD

**Discussion by micebrain** [Budibase/budibase#3192] [Ideas]
Hi,
I have several enums in my external Postgresql database. Is it possible for this fields with enum type to have dropdown selection instead of text in new record creation?
Also is it possible to configure certain field to have dropdown selection instead of plain text? E.g. dropdown with values like - Yes, Maybe, No.

**Comment by micebrain** [comment]
Oh.. i managed to figure out how to map to a dropdown field. It would be great to be able to select the enum for this.

**Reply by mjashanks** [reply]
Agreed, this would be a nice feature - will mark this as an idea.

**Reply by antimatter96** [reply]
> Oh.. i managed to figure out how to map to a dropdown field. It would be great to be able to select the enum for this.

Can you share ?

**Reply by micebrain** [reply]
@antimatter96 - I removed the default text field and added "Options Picker" from the Form list. Then on the configuration, there is a "Options source" setting. You can select "Custom" and then enter all the values.

**Reply by antimatter96** [reply]
Ohh, in the Design Tab, I was hoping in the Data Tab
